schutzbot/Jenkinsfile

```diff
@@ -164,6 +164,17 @@ pipeline {
                         }
                     }
                 }
+                stage('F32: koji-osbuild') {
+                    agent { label "f32cloudbase && x86_64" }
```

Comment:
Does it use stuff from RH internal repos or nested virt? If not, I suggest adding `&& aws` so we don't use more resource from PSI OpenStack than we really need. If it does, `&& psi` is needed.

Reply:
The Fedora tests should not need anything from the internal repos IIRC.

Reply:
Ah, I just removed the `psi` here (copied from the stage above), but didn't think to add `aws` in. Pushed a fix. Thanks!

Reply:
Thanks!